Math-sqrt-function.md

```diff
@@ -0,0 +1,34 @@
+# Math.sqrt() Function in Java
+**Description:** Math.sqrt() method returns the square root value of the primitive data type .If a number is less than zero, it will return NaN.
+## Syntax:
+     double sqrt(double d)
```

Comment:
Use multiline code syntax for Markdown

```java
double sqrt(double d)
```

And I think you mean signature. Not syntax.